docs/_config.yml

```diff
@@ -0,0 +1,88 @@
+# Hexo Configuration
+## Docs: http://hexo.io/docs/configuration.html
+## Source: https://github.com/hexojs/hexo/
+
+# Site
+title: Apollo Android Guide
+subtitle: Using Apollo with Android
+description:
+author:
+language:
+timezone:
+versions:
+  - '1'
+sidebar_categories:
+  null:
+    - index
+
+github_repo: apollographql/apollo-android
+content_root: docs/source
+
+social_links:
+  github: 'https://github.com/apollographql'
+  twitter: '@apollographql'
+
+# API keys
```

Comment:
Sanity check, not sure what this is exactly but  should api keys be checked into public repo?